docs/openapi.yaml

```diff
@@ -0,0 +1,1143 @@
+          required: false
+        - in: query
+          name: endTime
+          schema:
+            type: string
+            format: date-time
+            description: Maximum end time for returned reports
+          required: false
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Report"
+          description: Request successful
+        "404":
+          description: Not found - Mission object does not exist
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetails"
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetails"
+  /maps:
+    get:
+      operationId: getMaps
+      summary: Lookup all maps for asset
+      description: |
+        ### Overview
+        Lookup all maps for asset
+      tags:
+        - Maps
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Map"
+          description: Request successful
+        "404":
+          description: Not found - No map obejcts exist
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetails"
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetails"
+
+  /maps/{map_id}:
+    get:
+      operationId: getSingleMap
+      summary: Lookup map data for single map
+      description: |
+        ### Overview
+        Lookup a single map with the given id
+      tags:
+        - Maps
+      parameters:
+        - in: path
+          name: map_id
+          required: true
+          schema:
+            type: integer
+            example: 214
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Map"
+          description: Request successful
+        "404":
+          description: Not found - No map with given id exists
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetails"
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetails"
+
+  /reports:
+    get:
+      operationId: getReports
+      summary: Lookup all available reports on asset
+      description: |
+        ### Overview
+        Lookup all reports for performed missions on the asset.
```

Comment:
```suggestion 
        Lookup all reports for completed missions on the asset.
```

Reply:
Can we be more consistent in the use of `performed`/ `completed` and `active`/ `in_progress` and similar?